src/USER-MDI/mdi_init.h

```diff
@@ -0,0 +1,24 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+#ifndef LMP_INIT_MDI_H
+#define LMP_INIT_MDI_H
+
+#include "mdi.h"
+
+/* This is just a wrapper for the MDI Library's MDI_Init function, which simplifies
+   the process of compiling LAMMPS if the MDI Library has not been built.
+*/
+int init_mdi(const char* options, void* world_comm) { return MDI_Init(options, world_comm); };
```

Comment:
This wrapper doesn't really do much. What is it needed?

Reply:
As you say, it isn't necessary.  517f93d removes it.